changelog.json

```
@@ -1,5 +1,6 @@
 {
   "releases": {
+    "2.6.5": ["[Fixed] Crash sometimes when performing remote Git operations"],
```

```suggestion
    "2.6.5": ["[Fixed] Inform user of errors encountered when performing remote Git operations"],
```

---

Not sure what to think, I'm 👍  with improving the wording, but I think we shouldn't remove the "crash" part from the release note, since I'm guessing users will be looking for it to make sure that's what we fixed. What do you think? Maybe something like this?
```suggestion
    "2.6.5": ["[Fixed] Performing remote Git operations could crash the app"],
```

---

I like it!